test/langtools/jdk/javadoc/doclet/testLinkOption/TestRedirectLinks.java

```diff
@@ -66,12 +69,18 @@
 import toolbox.JavacTask;
 import toolbox.ToolBox;
 
+import jdk.test.lib.Platform;
+import jtreg.SkippedException;
```

Comment:
Nit: the order of imports on 72-73 needs to be swapped.

Reply:
Why? 'jdk' comes before 'jtreg' and 'Platform' comes before 'SkippedException'.
What am I missing here?

Reply:
Mild grumble: langtools tests do not rely on jdk test libraries

Reply:
Does langtools have its own test libraries that I can use to ask the same questions?

Reply:
Sorry, I was not clear.
The Fuzz.java has this order:
```
+import jdk.test.lib.Platform;
+import jtreg.SkippedException;
```
I thought, you ordered imports by names. Then it is better to keep this order unified.
It is really minor though.

Reply:
Sorry I'm still confused. As far as I can see, I've added the imports the
same way in both Fuzz.java and TestRedirectLinks.java.

And the imports are in sort order:
'jdk' comes before 'jtreg' and 'Platform' comes before 'SkippedException'.

Reply:
Sorry, copied fragment from a wrong file.
This file has imports out of the order:
  test/langtools/jdk/javadoc/doclet/testLinkOption/TestRedirectLinks.java

```
+ * @build jtreg.SkippedException
+ * @build jdk.test.lib.Platform
```